main.py

```diff
@@ -69,4 +69,4 @@ def sayNameFn(lst):
 
 a.sayName() # a is an instance of the Dog class
 Dog.saySpecies() # Dog is the class itself
-Dog.sayName()
\ No newline at end of file
+Dog.sayName() # This will throw an error
```

Comment:
```suggestion
Dog.sayName() # This will throw an error, my friend
```
do this instead it's more friendly